src/uct/base/uct_iface.c

```diff
@@ -540,10 +540,10 @@ UCS_CONFIG_DEFINE_ARRAY(alloc_methods, sizeof(uct_alloc_method_t),
                         UCS_CONFIG_TYPE_ENUM(uct_alloc_method_names));
 
 ucs_config_field_t uct_iface_config_table[] = {
-  {"MAX_BCOPY", "8192",
+  {"SEG_SIZE", "8192",
```

Comment:
i think we should remove SEG_SIZE from common code and push it down only to relevant transports only (e.g IB,MM)

Reply:
also, need to fix the description

Reply:
> i think we should remove SEG_SIZE from common code and push it down only to relevant transports only (e.g IB,MM)

yes, this is good point. done - now this is available for IB, MM, SELF, TCP transports (i.e. only for transports that depend on this value)